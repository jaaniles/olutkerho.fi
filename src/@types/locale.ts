export enum Locale {
  FI = "fi",
  EN = "en",
}

export const locales = Object.values(Locale);
