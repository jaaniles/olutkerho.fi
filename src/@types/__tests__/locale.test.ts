import { Locale, locales } from "../locale";

it("Check Locale", () => {
  expect.assertions(1);

  return expect(Locale).toMatchSnapshot();
});

it("Check locales", () => {
  expect.assertions(1);

  return expect(locales).toMatchSnapshot();
});
