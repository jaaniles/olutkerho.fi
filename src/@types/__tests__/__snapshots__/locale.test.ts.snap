// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Locale 1`] = `
Object {
  "EN": "en",
  "FI": "fi",
}
`;

exports[`Check locales 1`] = `
Array [
  "fi",
  "en",
]
`;
